src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -307,7 +307,7 @@ public function registerDoctrineMapping(array $config)
             'fieldName' => 'media',
             'targetEntity' => $config['class']['media'],
             'cascade' => [
-                 'persist',
```

Comment:
Please revert the formatting

Reply:
I think this one can stay @core23 , it changes from 5 to 4 spaces